docs/articles/documentation/using-testcafe/programming-interface/createtestcafe.md

```diff
@@ -9,20 +9,21 @@ checked: true
 Creates a [TestCafe](testcafe.md) server instance.
 
 ```text
-async createTestCafe([hostname], [port1], [port2]) → Promise<TestCafe>
+async createTestCafe([hostname], [port1], [port2], developmentMode) → Promise<TestCafe>
 ```
 
 Parameter                     | Type   | Description                                                                                                                                                                                                  | Default
 ----------------------------- | ------ | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------ | -------
 `hostname`&#160;*(optional)*       | String | The hostname or IP you will use to address the TestCafe server. Must resolve to the current machine. To test on external devices, use the hostname that is visible in the network shared with these devices. | Hostname of the OS. If the hostname does not resolve to the current machine - its network IP address.
 `port1`, `port2`&#160;*(optional)* | Number | Ports that will be used to serve tested webpages.                                                                                                                                                            | Free ports selected automatically.
+`developmentMode`&#160;*(optional)* | Boolean | Enables/disables mechanisms to log and diagnose errors. | `false`
 
 **Example**
 
 ```js
 const createTestCafe = require('testcafe');
 
-createTestCafe('localhost', 1337, 1338)
+createTestCafe('localhost', 1337, 1338, true)
```

Comment:
`createTestCafe('localhost', 1337, 1338, sslOptions, true)`

Reply:
Yes, we'll merge this string with a string from the PR about HTTPS